upa/test/censorshipClaimTest.ts

```diff
@@ -301,6 +319,17 @@ describe("Censorship challenge tests", () => {
             s2.computeProofDataMerkleProof(i)
           );
       }
+
+      const balance2 = await provider.getBalance(user1);
+
+      // Expect balance2 > balance 1
+      expect(balance2).is.greaterThan(balance1);
+      // Expect costAfterRefund < submitCost * 2% (we tolerate some minimal
+      // loss to the caller, but it should be very small compared to the
+      // overall cost)
+      const costAfterRefund = balance2 - user1StartBalance;
+      expect(costAfterRefund).is.lessThan(submitCost / 50n);
```

Comment:
Not sure how typescript handles division, but maybe is better:
```suggestion
      expect(costAfterRefund*50n).is.lessThan(submitCost);
```